dask/highlevelgraph.py

```diff
@@ -511,6 +511,7 @@ def layer_info_dict(self):
         info = {
             "layer_type": type(self).__name__,
             "is_materialized": self.is_materialized(),
+            "tasks (unoptimized)": f"{len(self)}",
```

Comment:
I think this will be number of layers not number of tasks. Right @ian-r-rose?

Reply:
I believe this is `Layer.__len__`, not `HighLevelGraph.__len__`, so it should be the right length for this part.